Revit_Core_Engine/Query/MEPConnectorsLocation.cs

```diff
@@ -0,0 +1,71 @@
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using Autodesk.Revit.DB;
+using BH.Engine.Adapters.Revit;
+using BH.oM.Adapters.Revit.Settings;
+using BH.oM.Reflection.Attributes;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Queries a Family Instance that contains MEP connectors for their locations.")]
+        [Input("mepInstance", "Revit family instance to be queried for their MEP connectors.")]
+        [Input("settings", "Revit adapter settings.")]
+        [Output("mepConnectorsLocation", "The list of points representing the connector's location points.")]
+        public static List<BH.oM.Geometry.Point> MEPConnectorsLocation(this FamilyInstance mepInstance, RevitSettings settings = null)
+        {
+            settings = settings.DefaultIfNull();
+            MEPModel mepModel = mepInstance.MEPModel;
+
+            if (mepModel == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("The input fitting does not have any connectors.");
+                return null;
+            }
+
+            ConnectorManager connectorManager = mepModel.ConnectorManager;
+            if (connectorManager == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("The input fitting does not have any connectors.");
+                return null;
+            }
+                    
+            ConnectorSet connectorSet = connectorManager.Connectors;
+            List<Connector> connectors = connectorSet.Cast<Connector>().ToList();
```

Comment:
`ToList()` seems to be redundant here - really minor though!

Reply:
Weirdly it still needs `ToList()`.

Reply:
Aye, you would need to change the variable type to `IEnumerable<T>`. But this is negligible, I like either way 👍